presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3667,10 +3667,10 @@ public void testExistsSubqueryWithGroupBy()
     @Test
     public void testCorrelatedScalarSubqueries()
     {
-        assertQuery("SELECT (SELECT n.nationkey) FROM nation n");
-        assertQuery("SELECT (SELECT 2 * n.nationkey) FROM nation n");
-        assertQuery("SELECT nationkey FROM nation n WHERE 2 = (SELECT 2 * n.nationkey)");
-        assertQuery("SELECT nationkey FROM nation n ORDER BY (SELECT 2 * n.nationkey)");
+        assertQuery("SELECT (SELECT n.nationkey + n.NATIONKEY) FROM nation n", "SELECT 2 * nationkey from nation");
+        assertQuery("SELECT (SELECT 2 * n.nationkey) FROM nation n", "SELECT 2 * nationkey FROM nation");
+        assertQuery("SELECT nationkey FROM nation n WHERE 2 = (SELECT 2 * n.nationkey)", "SELECT nationkey FROM nation WHERE nationkey = 1");
```

Comment:
why do these last 2 need comparison queries now?

Reply:
I think it' better to leave it out if the query shape is supported in h2 (no reason to introduce the opportunity for a bug in the expected query)

Reply:
Oh i forgot it works that way